.github/workflows/release_snapshot.yml

```diff
@@ -52,69 +51,47 @@ jobs:
            --operation=create
           source ./result.env
           echo "$CREATED_TAG" 
-          echo "created_tag=$CREATED_TAG" >> $GITHUB_OUTPUT
-  linux:
-    needs: linux-create
-    runs-on: ubuntu-latest
+          echo "created_tag=$CREATED_TAG" >> "$GITHUB_OUTPUT"
+          echo "created_tag=$CREATED_TAG" >> $GITHUB_ENV
+      - uses: ./.github/actions/test_install
+        with:
+          version_number: ${{ env.created_tag }}
+          python_version: "3.8"
+          os_platform: "linux"
+  build-for-os-and-python-versions:
+    needs: create-snapshot
     strategy:
       matrix:
         python-version: ["3.7", "3.9", "3.10"]
+        os: ["macos-latest", "ubuntu-latest"]
+        include:
+          - python-version: 3.8
+            os: macos-latest
     env:
       GH_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v3
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version }}
-      - run: |
+      - if: matrix.os == 'ubuntu-latest'
+        run: |
+          echo "install_os=linux" >> $GITHUB_ENV
```

Comment:
This field gets passed around and has 3 names: `os_platform`, `install_os` and `operating_system`.   Sticking with 1 will make it easier to follow it's the same field.